Zend/zend_compile.h

```diff
@@ -238,7 +238,7 @@ typedef struct _zend_oparray_context {
 /* or IS_CONSTANT_VISITED_MARK                            |     |     |     */
 #define ZEND_CLASS_CONST_IS_CASE         (1 << 6)  /*     |     |     |  X  */
 /*                                                        |     |     |     */
-/* Class Flags (unused: 30,31)                            |     |     |     */
+/* Class Flags (unused: 31).                              |     |     |     */
```

Comment:
Isn't flag 30 still unused? Whereas the 31 is used for strict types?

Reply:
I had some wonkery when fixing a merge conflict and I was quite tired. I'll revisit today with fresh eyes. Thanks for your review.